apps/src/blockly/addons/cdoTrashcan.js

```diff
@@ -167,24 +167,14 @@ export default class CdoTrashcan extends GoogleBlockly.DeleteArea {
         toolboxVisibility = 'hidden';
       }
 
-      /**
-       * NodeList.forEach() is not supported on IE. Use Array.prototype.forEach.call() as a workaround.
-       * https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach
-       */
-      Array.prototype.forEach.call(
-        // query selector for uncategorized toolbox contents
```

Comment:
Thanks for doing this! Lost a couple of comments here though that I think we should keep?

Reply:
Great catch!